processing.md

```diff
@@ -1162,189 +1162,174 @@ The processing archivist will:
-  </div>
-
-3\. _Medical records/records that document the medical history of individuals_ - The [Health Insurance Portability and Accountability Act (HIPAA)](https://www.hhs.gov/hipaa/) was enacted by the U.S. Congress in 1996. In accordance with the stipulations of HIPAA and other personal privacy regulations, access to the following types of records are restricted: formal medical examination forms, patient files or doctor’s notes on the condition of a patient and/or his/her treatment, autopsy records, medical histories, and correspondence regarding a specific individual’s medical state. Additional restricted materials include but are not limited to: medical insurance records or claims, personal injury, workers compensation records or claims, any documentation from an individual describing a medical situation (physical or mental illness, surgery), or similar information about others (spouse’s or child’s illnesses or sensitive circumstances surrounding their death). The HIPAA regulations are broad and far reaching. When processing or handling medical records please consult the Assistant Director for Processing and act cautiously to restrict sensitive materials.
-
-  - Example of standard RAC file-level access note -
-
-  <div class="docs-example">
-    <p>Restricted - confidential material.</p>
-  </div>
-
-4\. _Student records_ - [FERPA (the Family Education Right and Privacy Act)](http://www.ed.gov/policy/gen/guid/fpco/ferpa/index.html) restricts access to student records (e.g., health, grades, disciplinary action, etc.) for the lifetime of the student. FERPA protects student and parent rights to access their own student records, stored and maintained by the school, and restricts access to those records by others without the permission of the student or parents.
-
-The Buckley Amendment to FERPA allows directory information to be made public without permission, including the student name, address, telephone listing, date and place of birth, major field of study, participation in officially recognized activities and sports, weight and height of members of athletic teams, dates of attendance, degrees and awards received, and the most recent previous educational agency or institution attended by the student. Please consult the Assistant Director for Processing with any questions regarding the restriction of student records.
-
-  - Example of standard RAC file-level access note -
+### Step 24 - Add File-Level Conditions of Access Notes for Restricted Material
+The *Conditions Governing Access* note is used to document archival materials whose access is physically and/or intellectually restricted due to the nature of the information documented.
 
-  <div class="docs-example">
-    <p>Restricted - confidential material.</p>
-  </div>
+(For collection-level access statements please [see Step 7 of establishing DACS-Single Level Minimum Compliance at the Collection Level](https://docs.rockarch.org/processing-manual/planning#supplies))
```

Comment:
This link goes to step 7 in the manual which is about supplies rather than step 7 under DACS-single level min compliance about General Collection-Level Access Statements Approved for Use at RAC

Reply:
Thank you for catching that!!

Reply:
updated now!